frontend/src/lib/lemon-ui/LemonButton/LemonButtonLegacy.scss

```diff
@@ -0,0 +1,156 @@
+body:not(.posthog-3000) {
+    .LemonButton {
+        --lemon-button-height: 2.5rem;
+        min-height: var(--lemon-button-height);
+
+        &.LemonButton--has-side-icon {
+            padding-right: 0.5rem;
+        }
+
+        &.LemonButton--xsmall {
+            --lemon-button-height: 1.5rem;
+        }
+
+        &.LemonButton--small,
+        .Breadcrumbs3000 & {
+            --lemon-button-height: 2rem;
+            padding: 0.125rem 0.5rem;
+
+            &.LemonButton--has-icon:not(.LemonButton--no-padding),
+            &.LemonButton--no-content:not(.LemonButton--no-padding) {
+                padding-left: 0.375rem;
+            }
+
+            &.LemonButton--has-side-icon:not(.LemonButton--no-padding),
+            &.LemonButton--no-content:not(.LemonButton--no-padding) {
+                padding-right: 0.375rem;
+            }
+        }
+
+        &.LemonButton--full-width {
+            overflow: hidden;
+        }
+
+        &.LemonButton--large {
+            --lemon-button-height: 3.5rem;
+            gap: 0.75rem;
+            font-size: 1rem;
+
+            &.LemonButton--has-icon:not(.LemonButton--no-padding),
+            &.LemonButton--no-content:not(.LemonButton--no-padding),
+            &.LemonButton--has-side-icon:not(.LemonButton--no-padding) {
+                padding-right: 0.75rem;
+            }
+        }
+
+        // LemonStealth has some specific styles
+        &.LemonButton--status-stealth {
+            font-weight: 400;
+            color: var(--default);
+
+            &:not([aria-disabled='true']):hover,
+            &.LemonButton--active {
+                background: var(--primary-highlight);
+                color: inherit; // Avoid links being colored on hover
+            }
+
+            &.LemonButton--active {
+                font-weight: 500;
+
+                // These buttons keep their font-weight when actve
+                &.LemonButtonWithSideAction,
+                &[role='menuitem'],
+                &[aria-haspopup='true'] {
+                    font-weight: 400;
+                }
+            }
+
+            .LemonButton__icon {
+                color: var(--muted-alt);
+            }
+
+            // Secondary - outlined color style
+            &.LemonButton--secondary {
```

Comment:
<img width="265" alt="Screenshot 2023-11-15 at 17 59 52" src="https://github.com/PostHog/posthog/assets/2536520/f9578e99-d240-439d-b9b7-51a544f6373e">

Upside down buttons sometimes?

Reply:
Back to the old style <img width="285" alt="Screenshot 2023-11-16 at 14 43 53" src="https://github.com/PostHog/posthog/assets/6685876/d3e61661-a311-42a1-81ce-35611ae618b9">